tech/languages/python/python-installation.md

```diff
@@ -8,6 +8,28 @@ version: 2.7.10
 
 # Python Installation
 
-Python is installed by default in Fedora.
```

Comment:
Please still merge this line since both are really pre-installed now on the workstation. If that's not true for all Fedora variants, add "in Fedora Workstation".